pkg/operator/target_config_reconciler_v311_00.go

```diff
@@ -151,6 +162,34 @@ func managePod_v311_00_to_latest(client coreclientv1.ConfigMapsGetter, recorder
 	return resourceapply.ApplyConfigMap(client, recorder, configMap)
 }
 
+func checkForFeatureGates(featureGateLister configlistersv1.FeaturesLister) (map[string]bool, error) {
+	featureGateListConfig, err := featureGateLister.Get("cluster")
+	var featureGates = make(map[string]bool)
+	if apierrors.IsNotFound(err) {
```

Comment:
need to check for err that isn't a not found error as well

Reply:
Good catch, made the change. Thank you :)